cdap-ui/app/directives/node-metrics/node-metrics.less

```diff
@@ -22,7 +22,6 @@
   font-size: 11px;
   overflow: hidden;
   white-space: nowrap;
-  text-overflow: ellipsis;
```

Comment:
I think this shouldn't be removed. We should still show ellipses if the metrics still overflow even after we have removed the labels